### DISCUSSION TITLE
Nothing in SharpMonoInjectorCore

### DISCUSSION THREAD

**Discussion by Debouroute** [winstxnhdw/lc-hax#183] [Q&A] [answered]
I have nothing in SharpMonoInjectorCore is normal ??
and the cheat don't work

**Comment by winstxnhdw** [comment]
Of course, it's not normal. You need to recursively clone the repository.

**Reply by Debouroute** [reply]
What is the repository ?

**Reply by winstxnhdw** [reply]
I encourage you to read the README :)

**Reply by winstxnhdw** [reply]
You can read https://github.com/winstxnhdw/lc-hax/discussions/25 too.

**Reply by Debouroute** [reply]
what should I download ?

**Reply by winstxnhdw** [reply]
Everything under `Requirements`

**Reply by Debouroute** [reply]
can u just give SharpMonoInjectorCore folder pls ?

**Reply by winstxnhdw** [reply]
What's stopping me from giving you malware instead?

**Reply by Debouroute** [reply]
oh ok indeed please I beg you but without virus please

**Reply by winstxnhdw** [reply]
Sorry, but if you refuse to do even 1 minute worth of work, I don't think you are ready to use this.

**Reply by Debouroute** [reply]
I'm French, the translator translates anyhow so I don't understand anything

**Reply by winstxnhdw** [reply]
The kid in #25 is French too but he could do it.

**Reply by Debouroute** [reply]
why when I open with git bash here it puts me
"MyUser"@DESKTOP-AQGRR2M MINGW64 ~/Desktop/lc-hax-main/lc-hax
$
and if I follow the instructions I must type explorer.exe. in git bash cmd and I supposedly found lc-hax but there is nothing

**Reply by winstxnhdw** [reply]
Use the `git clone` command as shown in the README. Do not download the zip.

**Reply by Debouroute** [reply]
ok thx

**Reply by Debouroute** [reply]
where i download it if i don't download the zip file?

**Reply by winstxnhdw** [reply, marked as answer]
When you `git clone` as stated in the README, it will download it for you in the correct way.

**Reply by Debouroute** [reply]
My requierment don't work
![image](https://github.com/winstxnhdw/lc-hax/assets/155832054/3a506500-51ca-449b-a2a9-297446f0a81e)

**Reply by winstxnhdw** [reply]
That's because you didn't `git clone`. I keep telling you to `git clone` but you somehow ignored that for the last 4 times.

**Reply by Debouroute** [reply]
i have clone it
![image](https://github.com/winstxnhdw/lc-hax/assets/155832054/c3462621-168e-49a1-befd-edb1a3a73555)

**Reply by winstxnhdw** [reply]
Then open it and run requirements.

**Reply by Debouroute** [reply]
oh
![image](https://github.com/winstxnhdw/lc-hax/assets/155832054/c0497ed8-f219-4de6-89a5-caee78708290)

**Reply by winstxnhdw** [reply]
Did you download .NET 8 SDK?

**Reply by Debouroute** [reply]
ye i redownload for shure

**Reply by winstxnhdw** [reply]
Looks like you have the same problem as this guy: https://github.com/winstxnhdw/lc-hax/discussions/123

**Reply by Debouroute** [reply]
it's done i relauch the requierment but the same error are here

**Reply by winstxnhdw** [reply]
Nevermind, I think you just need to restart your computer.

**Reply by winstxnhdw** [reply]
If not, see this: https://stackoverflow.com/questions/68283730/error-nu1100-unable-to-resolve-microsoftofficecore-15-0-0-for-net5-0

**Reply by Debouroute** [reply]
what is the control for open the menu

**Reply by Debouroute** [reply]
i have that 
![image](https://github.com/winstxnhdw/lc-hax/assets/155832054/b7018db8-f25d-4850-b954-9437860f1f6a)
and i put it in plugin of bepinex

**Reply by winstxnhdw** [reply]
Please just use `launch.bat`. I won't troubleshoot using BepinEx.

**Reply by Debouroute** [reply]
that work but what is the control for open the menu

**Reply by Debouroute** [reply]
that work with the laucher.bat

**Reply by winstxnhdw** [reply]
There is no menu. Read the README.

**Reply by Debouroute** [reply]
how can i activate the ESP for example

**Reply by winstxnhdw** [reply]
ESP is activated by default.

**Reply by Debouroute** [reply]
oh and the anti kick

**Reply by winstxnhdw** [reply]
oh mon dieu

![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/2402a90a-7b57-46f0-8674-05ffcc33e814)

**Reply by Debouroute** [reply]
don't work but commands works

**Reply by winstxnhdw** [reply]
Only works in main menu
